python/setup.py

```diff
@@ -1,85 +1,38 @@
 import os
-import subprocess
-import shutil
 
-from setuptools import setup, Extension
-from setuptools.command.build_ext import build_ext
+from setuptools import setup
 from pip_correction import convert_to_manylinux
```

Comment:
You can import this from `cmake_setuptools` and remove the `pip_correction.py` file.